cmd/config.go

```diff
@@ -94,18 +94,22 @@ var configCmd = &cobra.Command{
 }
 
 func configCmdRun(cmd *cobra.Command, args []string) error {
-	var defaultAccountMark = "*"
+	var (
+		defaultAccountMark = promptui.Styler(promptui.FGBold, promptui.FGYellow)("*")
```

Comment:
```suggestion
		defaultAccountMark = promptui.Styler(promptui.FGYellow)("*")
```
When you put it in bold, it doesn't show the color.
I suggest you put either bold or yellow.

<img width="315" alt="Screenshot 2020-02-10 at 11 59 15" src="https://user-images.githubusercontent.com/16138690/74144573-4c788f80-4bfd-11ea-878e-d6fa2ff3e9c9.png">

Reply:
Good catch, thanks :)